src/javascript/app/pages/trade/price.js

```diff
@@ -118,6 +119,10 @@ const Price = (() => {
             proposal.barrier = parseInt(prediction.value);
         }
 
+        if (lots && isVisible(lots) && lots.value) {
+            proposal.amount = lots.value;
```

Comment:
can we combine this on line 56 were proposal.amount is being decided?

Reply:
I don't think those two blocks can be combined. I will just move it below the block on :56.